### ISSUE TITLE
Storage Access Framework

### ISSUE THREAD

**Issue by DarremMolko** [anilbeesetti/nextplayer#336]
Hi, I really love your video player. I noticed that it doesn't currently support accessing files stored in Document Providers. I was wondering if you would consider adding support for the Storage Access Framework to your app so users can easily access files from their preferred cloud storage providers. It would be a great addition to your already amazing app. Thanks!

**Comment by DarremMolko** [comment]
@anilbeesetti Hi, and sorry for the ping. I just tried a debug build with this commit https://github.com/anilbeesetti/nextplayer/commit/e8d062a88b216e015c42d6f4dfccd11fce826de3 and it works perfectly, but I noticed when I select the same video again it doesn't preserve the playback position, it starts from the beginning. Also how feasible is it to have a Resume fab to resume the last played media? Or maybe this is something unrelated and I should open a different issue.

**Comment by anilbeesetti** [comment]
Currently Next Player only preserve playback position of local media files. I will add remember playback position for other media in the future

**Comment by DarremMolko** [comment]
@anilbeesetti I investigated some more about this and I found we could mount and access entire directories, in fact mpv-android let's you do exactly that. It could be a Fab in the main screen like the current file picker or a setting in the "Media library" settings.
Here is the documentation:
https://developer.android.com/training/data-storage/shared/documents-files#grant-access-directory
https://developer.android.com/reference/android/content/Intent#ACTION_OPEN_DOCUMENT_TREE

By the way, congratulations for publishing the app to the Play Store, I gave you 5 stars.
